README.md

```diff
@@ -11,22 +11,23 @@ maps the regular CRS IDs from 900K for each rule to the range 9,900,000 - 9,999,
-| dos-protection-modsecurity-v3 | 9,515,000 - 9,515,999 | https://github.com/coreruleset/dos-protection-plugin-modsecurity-v3 | official | draft                | |
+| template                      | 9,500,000 - 9,500,999 | https://github.com/coreruleset/template-plugin                      | official  | tested               | [![.github/workflows/integration.yml](https://github.com/coreruleset/template-plugin/actions/workflows/integration.yml/badge.svg)](https://github.com/coreruleset/template-plugin/actions/workflows/integration.yml) |
+| auto-decoding                 | 9,501,000 - 9,501,999 | https://github.com/coreruleset/auto-decoding-plugin                 | official  | untested             | |
+| antivirus                     | 9,502,000 - 9,502,999 | https://github.com/coreruleset/antivirus-plugin                     | official  | &#9989;&nbsp;tested  | |
+| body-decompress               | 9,503,000 - 9,503,999 | https://github.com/coreruleset/body-decompress-plugin               | official  | &#9989;&nbsp;tested  | |
+| fake-bot                      | 9,504,000 - 9,504,999 | https://github.com/coreruleset/fake-bot-plugin                      | official  | &#9989;&nbsp;tested  | |
+| google-oauth2                 | 9,505,000 - 9,505,999 | https://github.com/coreruleset/google-oauth2-plugin                 | official  | &#9989;&nbsp;tested  | |
+| drupal-rule-exclusions        | 9,506,000 - 9,506,999 | https://github.com/coreruleset/drupal-rule-exclusions-plugin        | official  | untested             | |
+| wordpress-rule-exclusions     | 9,507,000 - 9,507,999 | https://github.com/coreruleset/wordpress-rule-exclusions-plugin     | official  | &#9989;&nbsp;tested  | |
+| nextcloud-rule-exclusions     | 9,508,000 - 9,508,999 | https://github.com/coreruleset/nextcloud-rule-exclusions-plugin     | official  | untested             | |
+| dokuwiki-rule-exclusions      | 9,509,000 - 9,509,999 | https://github.com/coreruleset/dokuwiki-rule-exclusions-plugin      | official  | untested             | |
+| cpanel-rule-exclusions        | 9,510,000 - 9,510,999 | https://github.com/coreruleset/cpanel-rule-exclusions-plugin        | official  | untested             | |
+| xenforo-rule-exclusions       | 9,511,000 - 9,511,999 | https://github.com/coreruleset/xenforo-rule-exclusions-plugin       | official  | &#9989;&nbsp;tested  | |
+| phpbb-rule-exclusions         | 9,512,000 - 9,512,999 | https://github.com/coreruleset/phpbb-rule-exclusions-plugin         | official  | &#9989;&nbsp;tested  | |
+| phpmyadmin-rule-exclusions    | 9,513,000 - 9,513,999 | https://github.com/coreruleset/phpmyadmin-rule-exclusions-plugin    | official  | being tested         | |
+| iRedMail-exclusions-plugin    | 9,518,000 - 9,518,999 | https://github.com/EsadCetiner/iRedMail-exclusions-plugin           | 3rd party | &#9989;&nbsp;tested  | |
```

Comment:
The table is ordered by rule IDs. Your entry should come after the performance plugin.

Reply:
done